tools/openshift-ci/Dockerfile

```diff
@@ -4,6 +4,15 @@ LABEL vendor="Red Hat inc."
 LABEL maintainer="OCP QE Team"
 USER root
 
+ARG YQ_VERSION="v4.30.8"
+ARG YQ_HASH="6c911103e0dcc54e2ba07e767d2d62bcfc77452b39ebaee45b1c46f062f4fd26"
+RUN set -x && \
+    YQ_URI="https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64" && \
```

Comment:
Soon, we'll need to work to have the arm images to execute tests on the arm-build01 prow cluster. 

without the integrity verification, [this is](https://github.com/openshift-qe/baremetal-qe-infra/blob/master/images/Dockerfile#L12-L14) how to avoid hard-coding the amd64 string in the download URL. If we want to maintain the integrity verification YQ_HASH has to become a bash associative array